src/sql/parts/notebook/models/cell.ts

```diff
@@ -222,11 +223,41 @@ export class CellModel implements ICellModel {
 		if (output) {
 			// deletes transient node in the serialized JSON
 			delete output['transient'];
-			this._outputs.push(output);
+			this._outputs.push(this.rewriteOutputUrls(output));
 			this.fireOutputsChanged();
 		}
 	}
 
+	private rewriteOutputUrls(output: nb.ICellOutput): nb.ICellOutput {
+		// Only rewrite if this is coming back during execution, not when loading from disk.
+		// A good approximation is that the model has a future (needed for execution)
+		if (this.future) {
+			try {
+				let result = output as nb.IDisplayResult;
+				if (result && result.data && result.data['text/html']) {
+					let nbm = (this as CellModel).options.notebook as NotebookModel;
```

Comment:
Cool, so we just missed porting this code? Was that the only issue / do other links work as expected?